xbmc/filesystem/VideoDatabaseDirectory/QueryParams.h

```diff
@@ -60,6 +62,8 @@ namespace XFILE
     long m_idSet;
     long m_idTag;
     long m_idVideoVersion{-1};
+    long m_videoAssetType{-1};
+    long m_idVideoAsset{-1};
```

Comment:
Just asking: in the mood to get rid of the magic numbers (which you have not introduced - I know)?